src/User/UserService.php

```diff
@@ -43,6 +55,17 @@ public function login(string $login, string $password): IdentityInterface
 
         $identity->resetToken();
         $this->identityRepository->save($identity);
+
+        $queueMessage = new QueueMessage(
```

Comment:
I think it should be a typed message instead:

```php
class UserLoggedInMessage extends AbstractMessage
{
    public function __construct(
         private string $id,
         private int $time
    )
    {
    }

    public function getHandlerName(): string
    {
        return LoggingAuthorizationHandler::NAME;
    }

    public function getData(): array
    {
        return [
            'id' => $this->id,
            'time' => $this->time,
        ];
    }
}
```

Reply:
done